src/components/QuestionPlay/WithoutOptions.tsx

```diff
@@ -20,6 +21,30 @@ export default function WithoutOptions({
   setIsAnswerRevealed,
   setSelectedChoices,
 }: Props) {
+  const [userResponse, setUserResponse] = useState({
```

Comment:
isConfirmed is confusing. Instead, we should just use isUndoTimerRunning boolean and store userResponse in a ref so that it can be submitted later.